### ISSUE TITLE
pushover_sound / pushover_silent addition?

### ISSUE THREAD

**Issue by michaelkrieger** [boredazfcuk/docker-icloudpd#124]
Can we add the ability to send pushover_sound=none to the script if set?
Pushover always has audio for me (due to certain 24/7 work notifications) so I'd like to just not be woken up with anything related to a background process which can be dealt with at a later point.

This could also be done with priority=-1  but just bypassing the sound should let the rest of it come together as it needs.

```
*priority* - send as -2 to generate no notification/alert, -1 to always send as a quiet notification, 1 to display as high-priority and bypass the user's quiet hours, or 2 to also require confirmation from the user
*sound* - the name of one of the sounds supported by device clients to override the user's default sound choice
```

**Comment by boredazfcuk** [comment]
Hi,

I've added the **pushover_sound** variable as Pushover, Prowl & Dingtalk notifications are triggered using the same command and priority is used by all of them. Also, different notifications are sent with different priorities.

**pushover_sound** must be set to one of the pre-defined sounds listed here: https://pushover.net/api#sounds

New container should be available on Dockerhub now. Let me know if you run into an problems.

Thanks.

**Comment by michaelkrieger** [comment]
You’re amazing. :)
I would have done the PR but figured I wanted to see how you’d want it implemented first. You beat me to it! Thank you!

closed with commit 513ba60

**Comment by boredazfcuk** [comment]
I did look at changing the priority as I thought that would be the better option. It would have meant splitting Pushover out into it's own section, in every place that sends a notification, so that it didn't affect Prowl and Dingtalk. As I'm using high priority notifications for failures and normal priority for standard messages, I'd have to have some way of overriding these defaults which would then have introduced two extra variables. Plus, I'd have looked to implement the same functionality to the other notification types... All of a sudden, there's a fair amount of changes needed and it's getting a little complicated.

Changing the sound was a quick win for yourself, but also, it adds a pretty decent feature for other users... It gives people a way to set an alternate ringtone for the notifications, so they know to ignore them without checking their device.

Thanks for the improvement.
